lib/helpers/app_logger.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:logger/logger.dart';
+
+final appLogger = Logger(
```

Comment:
@ahmed-tarek-salem please if you can add a short documentation here of the methods available